### ISSUE TITLE
Light mode down

### ISSUE THREAD

**Issue by nav1d** [ethereum/go-ethereum#15608]
Hi there,

please note that this is an issue tracker reserved for bug reports and feature requests.

For general questions please use the gitter channel or the Ethereum stack exchange at https://ethereum.stackexchange.com.

#### System information

Geth version: `1.7.3`
OS & Version: Windows 7-x64
Commit hash : (if `develop`)

#### Expected behaviour
finding at least one peer to update my blockchain.

#### Actual behaviour
won`t find peers on light mode to sync blockchain at all.
this week,seems network is somehow down.
due this issue, i can't transfer.

#### Backtrace

````
[backtrace]
````

**Comment by nav1d** [comment]
so am i alone?

**Comment by pepijnolivier** [comment]
@nav1d You are not alone, I just tried running light mode from geth 1.6.6-stable, which had worked fine about a month ago, but doesn't appear to sync right now...

I guess I'll try updating geth first

<img width="301" alt="1____library_application_support_ethereum_wallet_binaries_geth_unpacked__geth_" src="https://user-images.githubusercontent.com/2015108/33690793-87efdbd0-dae5-11e7-8ebe-986f83357663.png">

**Comment by pepijnolivier** [comment]
Might have something to do with the network being congested right now

**Comment by nav1d** [comment]
yeah,i think ethereum network does not have enough potential to handle clients.
i'm wonder what should we do with this shit?! if continues, it can freeze network.

**Comment by veox** [comment]
@nav1d This issue can not be solved with any code update. If people running "regular" `go-ethereum` nodes don't reserve any peer slots for "light" clients, then those will have nothing to connect to.

It _could_ perhaps be alleviated by implementing #15624, but that's a very niche case.

-----

@pepijnolivier Updating to a newer `geth` would help - [`v1.7.3`](https://github.com/ethereum/go-ethereum/releases/tag/v1.7.3) ideally, as it has LES `v2` support.

Note also that specifying `--syncmode light --maxpeers `**`100`** is exacerbating the issue. You're raising the default, and attempting to reserve more of this precious resource.

-----

Recommend closing as "nothing to do" - unless it can be demonstrated that a "light" node is rejected my a "lightserv" node that still has open peer slots.

**Comment by nav1d** [comment]
"nothing to do" !? 
bro, i know its about traffic and not about code. but according to my prev post, it should figured out to prevent, because this is about its structure.
downloding 30gb(fast mode) or 300gb(full mode) blockchain is not a good idea, many micro-trans made by smartphones and tablets. which smartphone can save 300gb into itself??
if ethereum wants to be used globally, it should reduce blockchain size or add more light-node peers.

**Comment by veox** [comment]
Yes, "nothing to do".

I'm in the same boat: my light clients are having trouble finding nodes to serve them. For that reason, I've started a dedicated public node, just to serve light clients (as mentioned in issue #15624, mentioned prior).

This is a _social_ issue. People don't enable "lightserv" mode. Tough luck.

-----

Now, if we were discussing the ramifications of enabling "lightserv" by default - that would be a different matter...

**Comment by shazow** [comment]
> This is a social issue. People don't enable "lightserv" mode. Tough luck.

Are there any public analytics on how many up-to-date "lightserv" nodes are online right now? Do we know what number of percentage is a healthy amount?

If there are numbers, we can start a campaign to increase it into a healthy amount.

> Now, if we were discussing the ramifications of enabling "lightserv" by default - that would be a different matter...

Is there an issue for this? (Should we open one?)

**Comment by veox** [comment]
@shazow Regarding a separate issue for `lightserv!=0` by default - I haven't found one (although there are probably several already). I wouldn't recommend doing this myself - see https://github.com/ethereum/go-ethereum/issues/15624#issuecomment-352181753.

-----

(#15454 seems to be the issue with the most laconic and on-point title.)

**Comment by veox** [comment]
IMO OK to close since v1.8.0 has been released.
